### ISSUE TITLE
C/S Error

### ISSUE THREAD

**Issue by hdoupe** [PSLmodels/OG-USA#549]
- inputs:
```json
{
    "meta_parameters": {
        "year": 2020,
        "data_source": "CPS",
        "time_path": true
    },
    "adjustment": {
        "Tax-Calculator Parameters": {},
        "OG-USA Parameters": {
            "world_int_rate": [
                {
                    "value": [
                        0.01
                    ]
                }
            ],
            "small_open": [
                {
                    "value": true
                }
            ],
            "delta_annual": [
                {
                    "value": 0.1
                }
            ],
            "Z": [
                {
                    "value": [
                        1,
                        1.015
                    ]
                }
            ],
            "epsilon": [
                {
                    "value": 0.7
                }
            ],
            "gamma": [
                {
                    "value": 0.4
                }
            ],
            "beta_annual": [
                {
                    "value": 0.98
                }
            ],
            "g_y_annual": [
                {
                    "value": 0.015
                }
            ],
            "frisch": [
                {
                    "value": 0.25
                }
            ]
        }
    }
}
```

- traceback:
```python
Traceback (most recent call last):
  File "/home/distributed/api/celery_app/__init__.py", line 89, in f
    outputs = func(*args, **kwargs)
  File "/home/distributed/api/celery_app/pslmodels_ogusa_tasks.py", line 58, in sim
    return functions.run_model(meta_param_dict, adjustment)
  File "/opt/conda/lib/python3.7/site-packages/cs_config/functions.py", line 208, in run_model
    base_tpi = TPI.run_TPI(base_params, client=client)
  File "/home/distributed/OG-USA/ogusa/TPI.py", line 432, in run_TPI
    K_init = firm.get_B(L_init, p.firm_r, p, 'TPI')
AttributeError: module 'ogusa.firm' has no attribute 'get_B'
```

**Comment by jdebacker** [comment]
@hdoupe Error with OG-USA.  Line 432 of `TPI.py` should be:
```
K_init = firm.get_K(L_init, p.firm_r, p, 'TPI')
```

Will push this fix to next PR.  Thanks for finding it.  We are not fully covered with tests of `TPI.py` with all permutations of the model because they take so long - but it bit us here after I used a replace all a while back.

**Comment by hdoupe** [comment]
Sounds good @jdebacker. Let me know when you're ready to update.

**Comment by jdebacker** [comment]
@hdoupe This should be fixed with PR #547.  I'm releasing a new version now.  Can you update on C/S?  Thanks!

**Comment by hdoupe** [comment]
Yep, starting the update now.

**Comment by hdoupe** [comment]
Update is live.

**Comment by jdebacker** [comment]
This has been fixed.  There is still a problem if on sets `small_open=True` and `zeta_K` to some number that is not one.  This problem is noted and would will be addressed in the solution to Issue #554.
